terraform/ecs/ecs.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_ecs_cluster" "ecscluster" {
+  name = "${var.projectname}-cluster"
+}
+
+resource "aws_launch_configuration" "grp3-lc" {
+  name = "${var.projectname}-lc"
+  image_id = "ami-064db566f79006111"
+  instance_type = var.inst_type
+  key_name      = var.inst_key
```

Comment:
make this variable not required as we don't really need a key for the instances (the idea is not need to jump in)

`key_name = var.inst_key != "" ? var.inst_key : ""`

That means that if the VAR has any values, use it, otherwise just leave empty